packages/cli/src/cmds/beacon/handler.ts

```diff
@@ -140,7 +145,7 @@ export async function beaconHandlerInit(args: BeaconArgs & GlobalArgs) {
   }
 
   // Create new PeerId everytime by default, unless peerIdFile is provided
-  const peerId = await createSecp256k1PeerId();
+  const peerId = args.peerIdFile ? await readPeerId(args.peerIdFile) : await createSecp256k1PeerId();
   const enr = SignableENR.createV4(createKeypairFromPeerId(peerId));
```

Comment:
I think we need something like:
```
// If peerIdFile
//   attempt to read enr from disk, beaconDir/enr
//   accept enr if enr.peerId == peerId
// else
//    create new enr
```

Otherwise, when using peerIdFile, then restarting, the discv5 network will ignore our enr updates since the enr seqno has been reset.